### DISCUSSION TITLE
Organisation-wide pipelines

### DISCUSSION THREAD

**Discussion by Gr3q** [woodpecker-ci/woodpecker#2280] [Request a Feature]
### Clear and concise description of the problem

It would be nice to add organisation-wide pipelines especially for some of the upcoming events (PR close, comment/issue events).

### Suggested solution

Ideally in the User/Organisation page there would be a section where it lists added pipelines with enable/disable options for said User/Organisation with options to add more or delete existing ones.

* **Add**: It's a repo picker with all repos in the organisation. You also provide the path to the pipeline inside the repo you want to use (in the default branch).

* **Delete**: You simply remove the pipeline with a delete button (deleting the pipeline should automatically remove it) 

* **Enable/Disable**: Instead of deletion you could temporarily disable added pipelines

Pipelines should work like any other pipeline. 

Prerequisites:
* It might be required to create new ENV variables but I don't really have a concrete list at this time.
* ~~Organisation-wide secrets would be preferable - Instance-wide secrets suffice in the meantime.~~

### Alternative

No alternative I can think of.

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://woodpecker-ci.org/docs/intro).
- [X] Check that there isn't already an [issue](https://github.com/woodpecker-ci/woodpecker/issues) that request the same feature to avoid creating a duplicate.

**Comment by patrickuhlmann** [comment]
Is there any possibility currently to define Instance-wide secrets?

**Comment by lafriks** [comment]
Both organization-wide and global secrets are already available in development (:next) version
